packages/core/components/FileDetails/FileAnnotationList.tsx

```diff
@@ -34,11 +34,15 @@ export default function FileAnnotationList(props: FileAnnotationListProps) {
 
         async function formatPathForHost() {
             if (!fileDetails || !active) return;
-
-            const localPath = fileDetails.localPath;
-            const path = localPath ? await executionEnvService.formatPathForHost(localPath) : null;
-
-            setLocalPath(path);
+            const localPath = fileDetails.getFirstAnnotationValue(AnnotationName.LOCAL_FILE_PATH);
+            if (!localPath) {
+                setLocalPath(null);
```

Comment:
Why set this to `null` here? It looks like if the boolean check on 43 fails we default to `null` anyway

Reply:
can't do exactly that because of the typing. Did something similar though in [55ab0e5](https://github.com/AllenInstitute/biofile-finder/pull/476/commits/55ab0e5082133b9c435eaef5fb5a52e2f1af5720)